andrada12/assignment4/WorldMap.java

```diff
@@ -0,0 +1,135 @@
+
+import java.util.ArrayList;
+import java.util.TreeSet;
+
+/*
```

Comment:
It's good practice to take out the IDE generated bolerplate comments (or set up your IDE to add useful comments)